model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -610,6 +632,9 @@ message StateResponse {
     // A response to clearing state.
     StateClearResponse clear = 1002;
   }
+
+  // Cache token which used to be part of the StateResponse
+  reserved 3;
```

Comment:
Not sure we need to do this since cache tokens were never set by the Runner/handled on the SDK side.

Reply:
Delete it.